File: Plovdiv/app/src/main/java/com/vesodeveloper/plovdiv/ContactActivity.java
package com.vesodeveloper.plovdiv;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class ContactActivity extends AppCompatActivity {
private Button ab;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_contact);
      ab=findViewById(R.id.ab);
      ab.setOnClickListener(new View.OnClickListener() {
          @Override
          public void onClick(View v) {
              Toast.makeText(getApplicationContext(),"vesohse@gmail.com",Toast.LENGTH_LONG).show();
          }
      });
    }

}

File: Sliven/app/src/main/java/com/vesodeveloper/sliven/ContactActivity.kt
package com.vesodeveloper.sliven

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast

class ContactActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_contact)
        var licensebtn=findViewById(R.id.licensebtn) as Button
        var appbtn=findViewById(R.id.appbtn) as Button
        appbtn.setOnClickListener {
            Toast.makeText(this,"Приложението е посветено на град Сливен. За контакт с програмиста: vrstanchev@gmail.com",Toast.LENGTH_LONG).show()
        }
        licensebtn.setOnClickListener {
            Toast.makeText(this,"Приложението е лицензирано под Свободния GNU GPL лиценз",Toast.LENGTH_LONG).show()
        }
    }
}